apps/docs/stories/components/checkout.stories.tsx

```diff
@@ -31,6 +31,50 @@ const meta: Meta = {
         category: 'props'
       }
     },
+    'disable-credit-card': {
+      type: 'boolean',
+      control: {
+        type: 'boolean',
+      },
+      description: 'Disable credit card payment method',
```

Comment:
Same here. What do you think about changing this to: `Disable new credit card payment method option`?